packages/backend/src/api/routes/health.ts

```diff
@@ -0,0 +1,24 @@
+import type { ApiRoutes } from '@vse-bude/shared';
+import { Router } from 'express';
+import { PrismaClient } from '@prisma/client';
+import { RedisStorageService } from 'services/redis-storage';
+
+export const initHealthRoutes = (path: ApiRoutes) => {
+  const router = Router();
+  let status = 200;
+
+  const prisma = new PrismaClient({
+    log: ['query', 'info', 'warn', 'error'],
+  });
+
+  try {
+    prisma.user.findMany();
+    new RedisStorageService();
```

Comment:
Use redisService, that is provided by initServices. Add seperate method to redisService `checkConnected`. It should make `client.ping` call and check whether `PONG` returned